.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/fcollonval/miniconda3/envs/widgets/bin/python"
```

Comment:
We could probably add `.vscode` (and others like `.ide`) to the `.gitignore`.

Reply:
Oups, sorry for than one

Reply:
no problem, thanks!